cmake/G3Test.h

```diff
@@ -89,7 +88,7 @@ inline void testEquivalence(const std::string& file, unsigned int line,
 /// TEST_GROUP(MyTests)
 #define TEST_GROUP(GROUPNAME) \
 namespace{ \
-static const char* GetTestGroup(){ return BOOST_PP_STRINGIZE(GROUPNAME); } \
+static const char* GetTestGroup(){ return #GROUPNAME; } \
```

Comment:
Direct use of the stringizing operator is not as general as `BOOST_PP_STRINGIZE`, as it does not allow other macros to be expanded. It can be replaced in two lines, though:

```
#define STRINGIZE(s) STRINGIZE_DIRECT(s)
#define STRINGIZE_DIRECT(s) #s
```